packages/core/src/telemetry/telemetry-utils.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ToolCallEvent } from './types.js';
+import { getLanguageFromFilePath } from '../utils/language-detection.js';
+
+export function addProgrammingLanguageToEvent(
```

Comment:
wrapping this doesn't fit right, consider making this an attribute of `gemini_cli.file.operation.count`